packages/json/src/serialize.ts

```diff
@@ -88,11 +88,11 @@ export function serializeProp(prop: ts.PropertySignature, checker: ts.TypeChecke
     const type = prop.type ? checker.typeToString(checker.getTypeFromTypeNode(prop.type)) : '';
 
     return {
-        name: prop.name.getText(),
+        name: prop.name ? prop.name.getText() : '*',
```

Comment:
Can you uncomment line 45 in file `examples/interfaces.ts`, remove the line 44 and run the tests?

You have to add the following code in `examples/interfaces.json` after line 81 to pass the tests
```
{
    "name": "*",
    "kind": "prop",
    "type": "any",
    "required": true,
    "readOnly": false,
    "description": ""
}
```

Reply:
Awesome ! Done